core/src/test/scala/com/pingcap/tispark/datasource/UpsertSuite.scala

```diff
@@ -58,15 +59,20 @@ class UpsertSuite extends BaseDataSourceTest("test_datasource_upsert") {
     testTiDBSelect(Seq(row2, row3, row4))
 
     // insert row2_v2 row5
-    tidbWrite(List(row2_v2, row5), schema)
-    testTiDBSelect(Seq(row2_v2, row3, row4, row5))
+    intercept[TiBatchWriteException] {
```

Comment:
change the file name? `UpsertSuite` -> `InsertSuite`?

Reply:
ok.